MAUI/DataGrid/sorting.md

```diff
@@ -0,0 +1,248 @@
+* **None** - None of the columns can be sorted.
+
+To apply to sort for multiple columns, tap the desired column headers after setting the `SfDataGrid.SortingMode` property to [Multiple](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridSelectionMode.html#Syncfusion_Maui_DataGrid_DataGridSelectionMode_Multiple).
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfDataGrid x:Name="sfDataGrid" SortingMode="Multiple" ItemsSource="{Binding OrderInfoCollection}" >
+
+    <syncfusion:SfDataGrid.SortColumnDescriptions>
+        <syncfusion:SortColumnDescription ColumnName="OrderID" SortDirection="Ascending" />
+    </syncfusion:SfDataGrid.SortColumnDescriptions>
+</syncfusion:SfDataGrid>
+{% endhighlight %}
+{% endtabs %}
+
+![Multi sorting .NET MAUI DataGrid](Images\sorting\maui-datagrid-multi-sorting.gif)
+
+## Tri-state sorting
+
+In addition, to sort the data in ascending/descending order, the `SfDataGrid` unsort the data in the original order by clicking the header again after sorting to descending order by setting the [SfDataGrid.AllowTriStateSorting](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_AllowTriStateSorting) property to true. When this property is set, sorting in each column iterates through three sort states: ascending, descending, and unsort.
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfDataGrid x:Name="sfDataGrid" SortingMode="Single" AllowTriStateSorting="True" ItemsSource="{Binding OrderInfoCollection}" >
+</syncfusion:SfDataGrid>
+{% endhighlight %}
+{% endtabs %}
+
+![Tri-state sorting .NET MAUI DataGrid](Images\sorting\maui-datagrid-tristate-sorting.gif)
+
+## Show sort number
+
+The `SfDataGrid` provides support to display the sequence numbers to denote the order of the column in which they are sorted during multiple columns sorting by setting [SfDataGrid.ShowSortNumbers](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_ShowSortNumbers) is set to true. This is applicable when the `SfDataGrid.SortingMode` property is `Multiple`.
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfDataGrid x:Name="sfDataGrid" SortingMode="Multiple" ShowSortNumbers="True" ItemsSource="{Binding OrderInfoCollection}" >
+</syncfusion:SfDataGrid>
+{% endhighlight %}
+{% endtabs %}
+
+![Sort with sequence numbers .NET MAUI DataGrid](Images\sorting\maui-datagrid-show-sort-numbers.png)
+
+## Sort column in double tap
+
+By default, column gets sorted when column header is clicked. This behavior can be changed to sort the column in double click action by setting [SfDataGrid.SortingGestureType](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_SortingGestureType) property to [DoubleTap](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridSortingGestureType.html#Syncfusion_Maui_DataGrid_DataGridSortingGestureType_DoubleTap).

```

Comment:
By default, the column gets sorted when the column header is clicked. This behavior can be changed to sort the column in a double-click action by setting the [SfDataGrid.SortingGestureType](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_SortingGestureType) property to [DoubleTap](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridSortingGestureType.html#Syncfusion_Maui_DataGrid_DataGridSortingGestureType_DoubleTap).

Reply:
Modified